Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ruby:2.3-slim
+MAINTAINER OpenStreetMap
+ENV REFRESHED_AT 2016-09-15
+
+# Install packages
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  build-essential \
+  ruby-dev \
+  libgdbm-dev \
+  libncurses5-dev \
+  libffi-dev \
+  libyaml-dev \
+  libreadline-dev \
+  libpq-dev \
+  libxml2-dev \
+  imagemagick \
+  libmagickwand-dev \
+  postgresql-client \
+  nodejs \
+  file
```

Comment:
You should add `&& apt-get clean && rm -rf /var/lib/apt/lists/` to avoid including cache files in the docker image, inflating its size.